crates/astria-composer/src/config.rs

```diff
@@ -41,11 +41,8 @@ pub struct Config {
     /// Address of the RPC server for the sequencer chain
     pub sequencer_url: String,
 
-    /// Chain ID that we want to connect to
-    pub chain_id: String,
-
-    /// Address of the RPC server for execution
-    pub execution_url: String,
+    /// A list of <chain_id>::<url> pairs
+    pub rollups: String,
```

Comment:
Should be a `Vec<Rollup>` imo, see comment on `Searcher::from_config()`

Reply:
Similar argument to above: for hygiene reasons I prefer to leave `Rollup` private to the `searcher` module and not make it crate-private, which prevents this.